controller/schema.go

```diff
@@ -241,6 +241,45 @@ $$ LANGUAGE plpgsql`,
 	migrations.Add(10,
 		`ALTER TABLE formations ADD COLUMN tags jsonb`,
 	)
+	migrations.Add(11,
+		`ALTER TABLE artifacts ADD COLUMN attributes jsonb`,
+		`CREATE TABLE release_tar_artifacts (
```

Comment:
Should this just be `release_artifacts`? Is there a reason to specialize it for tarballs? We could enforce the "base image" requirement in the database and move the artifact field out of the release and into this join table.